### PR TITLE
Get Items Endpoint

### PR DESCRIPTION
**Pull request by Mackenzie-Frey**
### This Pull Request Implements:
- 200 response with a list of items associated with the business's distributor. The business is known via an API key passed in. If the API key does not match a 401 Unauthorized is returned.
- Adds gem for Fast JSON API
- Adds methods for `find_business(api_key)`, and `unauthorized` to `ApplicationController`, let's be sure to use these as we continue coding.
- Model method and testing for `find_distributor(api_key)`.
	
- [X] Linked To User Story on Trello

### This Pull Request Does Not:
Utilize the initial sorting by `alc_type` and `alc_category`, that we had originally planned for. The level of difficulty in doing this/time required is greater than we expected due to an array of model objects needing to be serialized, need to sort these by type and category, and display the items in these nested, sorted categories. Mike and I spoke and evaluated it was not a priority to continue on this route for the back-end team, as the front-end could parse through the data in an expected low amount of time and difficulties.

### Applicable Images
image

image

image

### CODE REVIEW

**Comment by Mackenzie-Frey** [comment]
[Distributor items endpoint](https://trello.com/c/geL8vqk4/5-distributor-items-endpoint)

File: app/models/item.rb [hunk 1]
```diff
@@ -22,4 +22,8 @@ class Item < ApplicationRecord
     only_integer: true,
     greater_than_or_equal_to: 1
   }
+
+  def self.distributor_items(distributor_id)
```
Comment (juliamarco) [inline app/models/item.rb]: Looking good!

File: spec/requests/api/v1/items_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe 'Items API', :type => :request do
+  context 'with a correct API key' do
+    it 'sends a list of items for a successful request' do
+      distributor_1 = Distributor.create!(name: 'RNDC',
+                                        address: '3319 Arapahoe st, Denver, CO',
```
Comment (juliamarco) [inline spec/requests/api/v1/items_request_spec.rb]: I think this needs to be indented (also applies for the other tests)

File: spec/requests/api/v1/items_request_spec.rb [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      expect(result[0]['attributes']['name']).to eq(item_1.name)
+      expect(result[0]['attributes']['price']).to eq(item_1.price)
+      expect(result[0]['attributes']['ounces']).to eq(item_1.ounces)
+      expect(result[0]['attributes']['unit']).to eq(item_1.unit)
+      expect(result[0]['attributes']['thumbnail']).to eq(item_1.thumbnail)
+    end
+  end
+
+  context 'with an incorrect API key' do
+    it 'responds with a 401 unauthorized status' do
```
Comment (juliamarco) [inline spec/requests/api/v1/items_request_spec.rb]: We should change this to a 404 now...

**Pull request merged**